components/gallery/GalleryItem.vue

```diff
@@ -212,9 +227,34 @@ useNuxt2Meta({
   }
 }
 
-.gallery-item-media image {
-  margin-left: auto;
-  margin-right: auto;
+.fullscreen-button {
+  position: absolute;
+  right: 2.75rem;
+  top: 2rem;
+  z-index: 1;
+  display: none;
+  width: 35px;
+  height: 35px;
+  border: 1px solid;
+  color: #000 !important;
+  background: rgba(255, 255, 255, 0.15);
+  border-color: rgba(255, 255, 255, 0.3);
+}
+
+.dark-mode .fullscreen-button {
+  color: #fff !important;
+  background: rgba(0, 0, 0, 0.15);
+  border-color: rgba(0, 0, 0, 0.3);
+}
```

Comment:
check out `@include ktheme()` for theming color, and if possible try to avoid using `!important`